### PR TITLE
RailroadStation  implementation

### PR DESCRIPTION
**Pull request by janosikm**

### CODE REVIEW

File: server/game/cards/01-DTR/RailroadStation.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            },
+            handler: context => {
+                this.game.promptForLocation(context.player, {
+                    activePromptTitle: 'Select destination',
+                    waitingPromptTitle: 'Waiting for opponent to select location',
+                    onSelect: (player, location) => {
+                        this.game.resolveGameAction(GameActions.moveDude({ 
+                            card: context.target, 
+                            targetUuid: location.uuid, 
+                            options: { needToBoot: false, allowBooted: true }
```
Comment (mmeldo) [inline server/game/cards/01-DTR/RailroadStation.js:23]: Nice 😄 Your first card, but I hope you understand I cannot just let it without any comment 😛 The `options` can be removed as those are default. Just to make it simpler.

**Review by mmeldo — COMMENTED** [review]
🥳 parada kamo

File: server/game/cards/01-DTR/RailroadStation.js (hunk 1 above)
Comment (janosikm) [reply server/game/cards/01-DTR/RailroadStation.js]: after some struggle ... it's gone :)

**Review by mmeldo — APPROVED** [review]
juchu 🎉

**Pull request merged**